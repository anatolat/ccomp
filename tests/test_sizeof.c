
int main() {
	char ch;
	int i;
	char *ptr;

	printf("%d %d %d %d %d\n", sizeof(i), sizeof(ch), sizeof(ptr), sizeof(*ptr), sizeof(ptr[0]));

	char str[256];
	printf("%d %d\n", sizeof(str), sizeof(str[0]));
}