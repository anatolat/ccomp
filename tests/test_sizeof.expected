4 1 4 1 1
256 1
